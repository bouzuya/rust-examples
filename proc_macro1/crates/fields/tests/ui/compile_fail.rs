fn main() {
    fields::fields!(field1 = "abc");
}
