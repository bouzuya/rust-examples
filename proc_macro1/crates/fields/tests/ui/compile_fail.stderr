error: expected one of: `for`, parentheses, `fn`, `unsafe`, `extern`, identifier, `::`, `<`, `dyn`, square brackets, `*`, `&`, `!`, `impl`, `_`, lifetime
 --> tests/ui/compile_fail.rs:2:30
  |
2 |     fields::fields!(field1 = "abc");
  |                              ^^^^^
